packages/nc-gui/components/cell/attachment/Image.vue

```diff
@@ -0,0 +1,25 @@
+<script setup lang="ts">
+interface Props {
+  src: string[]
+  alt?: string
+}
+
+const props = defineProps<Props>()
+
+const index = ref(0)
+
+const onError = () => index.value++
+</script>
+
+<template>
+  <LazyNuxtImg
+    v-if="index < props.src.length"
+    class="m-auto"
+    :src="props.src[index]"
+    :onerror="onError"
```

Comment:
You can replace it with `@error="onError"`

Reply:
Thanks. Updated.